src/Frame.h

```diff
@@ -0,0 +1,35 @@
+#ifndef __FRAME_H__
+#define __FRAME_H__
+
+#include <string>
+#include <cstdint>
+
+using namespace std;
+
+class Frame {
+public:
+    // Getters
```

Comment:
Is there a need to modify a `Frame` object at runtime? If not, this could be done more cleanly with initializer lists.